lib/train/file/remote/windows.rb

```diff
@@ -77,6 +77,10 @@ def link_path
           nil
         end
 
+        def shallow_link_path
```

Comment:
I'm no expert, but my understanding is that Win10 does support symbolic links.

Reply:
Indeed, that's why we have a nil (same as link_path above)

Reply:
What I mean to say is that because Win10 supports symbolic links, I would expect the implementation of link_path and shallow_link_path to be able to handle a symbolic link on windows 10, and return nil on other versions of windows that do not support symlinks.